scene/resources/particles_material.h

```diff
@@ -92,7 +101,7 @@ class ParticlesMaterial : public Material {
 
 private:
 	union MaterialKey {
-		// The bit size of the struct must be kept below or equal to 32 bits.
+		// The bit size of the struct must be kept below or equal to 64 bits.
```

Comment:
Was there a good reason to keep this below 32? We might be breaking something here.

Reply:
Honestly, I can't answer that - maybe @reduz can give a more in depth answer?

Reply:
Also @clayjohn

Reply:
I think, if I didn't miscalculate... after changing the mode from the int to bool, it should be exactly inside the 32 bits. The easiest solution to get this PR through is to change the comment back to 32 bits and then do the neccessary changes outside of this PR.